src/libmongoc/src/mongoc/mongoc-crypto-common-crypto.c

```diff
@@ -20,7 +20,21 @@
 #include "mongoc-crypto-common-crypto-private.h"
 #include <CommonCrypto/CommonHMAC.h>
 #include <CommonCrypto/CommonDigest.h>
+#include <CommonCrypto/CommonKeyDerivation.h>
 
+bool
+mongoc_crypto_common_crypto_pbkdf2_hmac_sha1 (mongoc_crypto_t *crypto,
+                                              const char *password,
+                                              size_t password_len,
+                                              const uint8_t *salt,
+                                              size_t salt_len,
+                                              uint32_t iterations,
+                                              size_t output_len,
+                                              unsigned char *output)
+{
+   return !CCKeyDerivationPBKDF (
```

Comment:
```suggestion
   return kCCSuccess == kCCKeyDerivationPBKDF (
```

Suggest checking returned `int` to avoid unchecked cast to bool. The error codes [are defined as negatives](https://github.com/apple-oss-distributions/CommonCrypto/blob/CommonCrypto-60178.40.2/include/CommonCryptoError.h#L57). Add `#include <CommonCrypto/CommonCryptoError.h>` to include definition for `kCCSuccess`.